lms/djangoapps/course_api/blocks/tests/test_forms.py

```diff
@@ -100,8 +102,22 @@ def test_non_existent_usage_key(self):
 
     #-- user
 
-    def test_no_user_param(self):
+    @ddt.data("True", "true", True, "False", "false", False, None)
+    def test_no_user_staff(self, all_blocks_value):
```

Comment:
All the conditionals in this test make it a bit confusing for me.  I'd like to suggest breaking it apart into three tests--something like: `test_all_blocks_true`, `test_all_blocks_false`, and `test_all_blocks_none`.